node/service/src/lib.rs

```diff
@@ -69,28 +69,40 @@ native_executor_instance!(
 	pub MoonbeamExecutor,
 	moonbeam_runtime::api::dispatch,
 	moonbeam_runtime::native_version,
-	frame_benchmarking::benchmarking::HostFunctions,
+	(
+		frame_benchmarking::benchmarking::HostFunctions,
+		moonbeam_primitives_ext::moonbeam_ext::HostFunctions
+	),
 );
 
 native_executor_instance!(
 	pub MoonriverExecutor,
 	moonriver_runtime::api::dispatch,
 	moonriver_runtime::native_version,
-	frame_benchmarking::benchmarking::HostFunctions,
+	(
+		frame_benchmarking::benchmarking::HostFunctions,
+		moonbeam_primitives_ext::moonbeam_ext::HostFunctions
+	),
 );
```

Comment:
Is tracing functionality compiled into all nodes, or is there a feature flag? If there is a feature flag, we should use it here.

Reply:
It's compiled for all nodes and enabled through a cli flag.